src/ImageSharp/Image/PixelArea{TColor}.cs

```diff
@@ -146,34 +168,48 @@ public PixelArea(int width, int height, ComponentOrder componentOrder, int paddi
         public byte[] Bytes { get; }
 
         /// <summary>
+        /// Gets the component order.
+        /// </summary>
+        public ComponentOrder ComponentOrder { get; }
+
+        /// <summary>
         /// Gets the pointer to the pixel buffer.
         /// </summary>
         public IntPtr DataPointer => this.dataPointer;
 
         /// <summary>
+        /// Gets the height.
+        /// </summary>
+        public int Height { get; }
+
+        /// <summary>
         /// Gets the data pointer.
         /// </summary>
         public byte* PixelBase { get; private set; }
 
         /// <summary>
-        /// Gets the component order.
+        /// Gets number of bytes in a row.
         /// </summary>
-        public ComponentOrder ComponentOrder { get; }
+        public int RowByteCount { get; }
 
         /// <summary>
         /// Gets the width.
         /// </summary>
         public int Width { get; }
 
         /// <summary>
-        /// Gets the height.
+        /// Gets the pool used to rent <see cref="Bytes"/>, when it's not coming from an external source
         /// </summary>
-        public int Height { get; }
+        // ReSharper disable once StaticMemberInGenericType
+        private static ArrayPool<byte> BytesPool => ArrayPool<byte>.Shared;
```

Comment:
I'm afraid, using `ArrayPool<T>.Shared` is not the best for this library. Sample issue: What if you exceed the maximum size with large images?
I used it here because I was "lazy" to find the best solution for `PixelArea˛`. In such cases defining an extra indirection level could be useful for a future refactor. I will leave a comment about this.